adwords_client/client.py

```diff
@@ -257,9 +281,9 @@ def _close_file(file_handler):
             file_handler.close()
 
         with ThreadPoolExecutor() as executor:
-            executor.map(_close_file, self._open_files.values())
+            executor.map(_close_file, self.open_files.values())
 
-        self._open_files.clear()
+        self.open_files.clear()
```

Comment:
Not that we should change it, but I wish it to be named `files_open` or `files_opened` instead of `open_files`...